src/native/eventpipe/ep-event-instance.c

```diff
@@ -69,7 +69,7 @@ ep_event_instance_init (
 
 #ifdef EP_CHECKED_BUILD
 	event_instance->debug_event_start = 0xDEADBEEF;
-	event_instance->debug_event_end = 0xCAFEBABE;
+	event_instance->debug_event_end = 0x0DDBA11;
```

Comment:
Aligning to some other convention on how to mark these things?

Reply:
No convention; `0xCAFEBABE` just felt a little anachronistic to modern sensibilities. It is occurring to me that `0DB411` is only 7 bytes for the "word" which is bugging me, so I'll find another 8 byte "word" to use.

Reply:
Agree, an maybe we should try to set high bits as well in whatever we choose, do they really need to be different, couldn't we use DEADBEEF on both.